datahost-ld-openapi/src/tpximpact/datahost/ldapi/handlers.clj

```diff
@@ -0,0 +1,26 @@
+(ns tpximpact.datahost.ldapi.handlers
+  (:require
+   [tpximpact.datahost.ldapi.series :as series]))
+
+(defn get-dataset-series [db {{:keys [series-slug]} :path-params}]
+  (let [key (series/dataset-series-key series-slug)
+        jsonld-doc (get @db key)]
+    (if jsonld-doc
+      {:status 200
+       :body jsonld-doc}
+      {:status 404
+       :body "Not found"})))
+
+(defn put-dataset-series [db {:keys [body-params path-params query-params]}]
+  (let [params (-> query-params (update-keys keyword) (merge path-params))]
+    (try
+      (swap! db series/upsert-series {:api-params params
+                                      :jsonld-doc body-params})
+      {:status 200
```

Comment:
status should be `201` for case where it's created -- return `:created` or `:updated` from `upsert-series` and handle here